keyboards/hs60/v1/info.json

```
@@ -0,0 +1,84 @@
+{
+  "keyboard_name": "HS60",
+  "manufacturer": "Yiancar-Designs",
+  "url": "",
+  "maintainer": "qmk",
+  "usb": {
+    "vid": "0xFEED",
+    "pid": "0x0257",
+    "device_version": "0.0.1"
+  },
+  "rgb_matrix": {
+    "animations": {
+      "alphas_mods": true,
+      "gradient_up_down": true,
+      "gradient_left_right": true,
+      "breathing": true,
+      "band_sat": true,
+      "band_val": true,
+      "band_pinwheel_sat": true,
+      "band_pinwheel_val": true,
+      "band_spiral_sat": true,
+      "band_spiral_val": true,
+      "cycle_all": true,
+      "cycle_left_right": true,
+      "cycle_up_down": true,
+      "rainbow_moving_chevron": true,
+      "cycle_out_in": true,
+      "cycle_out_in_dual": true,
+      "cycle_pinwheel": true,
+      "cycle_spiral": true,
+      "dual_beacon": true,
+      "rainbow_beacon": true,
+      "rainbow_pinwheels": true,
+      "raindrops": true,
+      "jellybean_raindrops": true,
+      "hue_breathing": true,
+      "hue_pendulum": true,
+      "hue_wave": true,
+      "pixel_rain": true,
+      "pixel_flow": true,
+      "pixel_fractal": true,
+      "typing_heatmap": true,
+      "digital_rain": true,
+      "solid_reactive_simple": true,
+      "solid_reactive": true,
+      "solid_reactive_wide": true,
+      "solid_reactive_multiwide": true,
+      "solid_reactive_cross": true,
+      "solid_reactive_multicross": true,
+      "solid_reactive_nexus": true,
+      "solid_reactive_multinexus": true,
+      "splash": true,
+      "multisplash": true,
+      "solid_splash": true,
+      "solid_multisplash": true
+    },
+    "driver": "is31fl3731",
+    "max_brightness": 215
```

Noticed this while removing the other `suspend_power_down_kb` instances, it would currently be doing nothing without sleep define. However i think the existence of the `suspend_power_down_kb` code somewhat implies the intention to actually sleep RGB
```suggestion
    "max_brightness": 215,
    "sleep": true
```